### ISSUE TITLE
Fix constants in endpoint urls.

### ISSUE THREAD

**Issue by tyte** [karina-klinkeviciute/rainbow-challenge-front#47]

**Comment by Colin-Stark** [comment]
There's a database you could use with only one endpoint and it's really easy to maneuver and get your required data

**Comment by Colin-Stark** [comment]
I could do a sample of your rainbow challenge front app in it for you to see

**Comment by karina-klinkeviciute** [comment]
> 

@Colin-Stark we already have a backend, it's built with Django. It's a REST API backend. Is the one you're suggesting using GraphQL?

**Comment by Colin-Stark** [comment]
It's a rest API cloud database using SQL and NoSQL

It's called Harperdb. It is pretty easy to integrate but you already have done that

**Comment by karina-klinkeviciute** [comment]
@Colin-Stark yes, we have a backend with Django and Python, the code for it is here if you're interested: https://github.com/karina-klinkeviciute/rainbow-challenge/projects/1
